src/lib.rs

```diff
@@ -131,6 +131,7 @@ pub(crate) mod private {
     pub trait Sealed {}
 }
 
+pub mod nonblocking;
```

Comment:
```suggestion
#[cfg(feature = "nonblocking")]
pub mod nonblocking;
```
IMO it's easier to notice conditional compilation when using outer attributes, not inner ones; it feels nicer to have them all in one place.